### ISSUE TITLE
iOS not loading image -> <error>: CGAffineTransformInvert: singular matrix.

### ISSUE THREAD

**Issue by Axort** [alwx/react-native-photo-view#87]
Hi!

Sometimes In iOS the images doesn't load. In Android works perfectly.

I'm using:
react-native: 0.45.1
react-native-photo-view@1.4.0

This is my code:

```
<PhotoView 
                source={{uri: this.props.photo.url}} 
                minimumZoomScale={1}
                maximumZoomScale={3}
                androidScaleType='fitCenter'
                onLoad={() => this.setState({ imageLoading: false })}
                onLoadStart={() => console.log('download started')}
                onLoadEnd={() => console.log('download ended')}
                style={styles.imageFullScreen} />
```

I noticed that when the images don't load, the console show this error:

**<Error>: CGAffineTransformInvert: singular matrix.**

Also when the image loads successfully, the 'download started' console log appears. When the image fail to load, it only shows the 'download ended' console log.

Am i doing something wrong? or is there a property am I missing?

Thanks in advance.

**Comment by samanthacampisi** [comment]
I can confirm this

**Comment by zackify** [comment]
Not seeing the error, but every few times you view an image it will return a black area after onLoadEnd

**Comment by srameshr** [comment]
Yes! Same here. I can confirm this. I dont see any logs in the simulator logger when the image fails to load. Just a blank screen. My images are being served over https.
Did anyone find a solution for this?

**Comment by igMartin** [comment]
Same issue here guys. I discover that if I change the image, it would load perfectly the first time. If I navigate, changing route and go back (or either refresh the app), it will show a black area. If I change the image for a new one, it will work again (until I change route or reload, then the black box again.) Couldn't find a solution yet.

**Comment by srameshr** [comment]
@igMartin Same here. Both on 1.3.x and 1.4.x

**Comment by fschai89** [comment]
I also face same problem, the image sometimes load sometimes not load in IOS.
